### ISSUE TITLE
Bad Memory Error

### ISSUE THREAD

**Issue by ParthShah412** [parlance/ctcdecode#97]
When I am running the ctc decode, I am encountering the following error... I am not sure what is wrong

    label_map = [' ']+PHONEME_MAP
    decoder = CTCBeamDecoder(labels=label_map,blank_id=0)
    for (batch_num,collate_output) in enumerate(data_loader):
        print(batch_num)
        data,cat_output,len_inp,len_otp = collate_output
        data = data.to(device)
        output = model(data,len_inp)
        len_otp_tensor = torch.tensor(np.asarray(len_otp))

        output = output.view(-1,batch_size,output.shape[1])

        output_log = output.log_softmax(2).detach().requires_grad_()
        output, scores, timesteps, out_seq_len = decoder.decode(probs=output_log, seq_lens=len_otp_tensor)

File "/home/ubuntu/anaconda3/envs/pytorch_p36/lib/python3.6/site-packages/ctcd ecode/__init__.py", line 40, in decode  output, timesteps, scores, out_seq_len)
--
